src/SessionSetup/ConnectToStadiaWidget.cpp

```diff
@@ -91,6 +94,14 @@ ConnectToStadiaWidget::ConnectToStadiaWidget(QWidget* parent)
                    [this]() { emit InstanceReloadRequested(); });
 
   SetupStateMachine();
+
+  if (absl::GetFlag(FLAGS_enable_project_selection)) {
+    ui_->instancesFilterLineEdit->hide();
+    ui_->refreshButton->hide();
+  } else {
+    // hide() does not work on QLayouts, hence its removed from the parent
```

Comment:
```suggestion
    // hide() does not work on QLayouts, hence it's removed from the parent
```

Reply:
changed to comment, so it doesn't apply anymore